### ISSUE TITLE
Logs truncated to 500 lines

### ISSUE THREAD

**Issue by mackoftrack** [spring-cloud/spring-cloud-dataflow#3679]
**Description:**
Task execution logs show only the last 500 lines of log

**Release versions:**
2.2.2.RELEASE

**Steps to reproduce:**
- Launch a task
- View task execution logs

**Screenshots:**

![image](https://user-images.githubusercontent.com/3811209/70262106-8f733e00-1748-11ea-8303-bfd0376965e5.png)

**Comment by ilayaperumalg** [comment]
@mackoftrack I assume you are using K8s deployer. Currently, we hard code the tail lines of logs to `500` when showing the log of a `Pod` as in [here](https://github.com/spring-cloud/spring-cloud-deployer-kubernetes/blob/master/src/main/java/org/springframework/cloud/deployer/spi/kubernetes/KubernetesAppDeployer.java#L244). We'll enhance this with some configuration options in the upcoming releases.

**Comment by mackoftrack** [comment]
@ilayaperumalg Yes using k8s deployer. Thanks for explanation. This should definitely be configurable, or at least default to showing all logs perhaps? Why limit to 500 lines?

**Comment by ilayaperumalg** [comment]
We wanted to have a reasonable default for the initial implementation and hence `500` being the current default. Yes, we'll have the configuration options in the deployer implementations. I am moving this to K8s deployer to track it from there. Thanks for the inputs.
